py/src/ai/h2o/sparkling/ml/params/H2OCommonSupervisedParams.py

```diff
@@ -23,17 +23,59 @@
 
 
 class H2OCommonSupervisedParams(H2OCommonParams, H2OSupervisedMOJOParams):
+    foldCol = Param(
```

Comment:
The duplication of params with H2OCommonUnsupervisedParams.py will be removed after we start generating python if I understand correctly? At this stage, we just prepared H2OCommonParams.py so it will remain as it is.

Reply:
Yes, exactly. I just wanted to do a minimal change to make python tests to pass. When python code gets generated, there won't be nieghter `H2OCommonUnsupervisedParams` or `H2OCommonSupervisedParams`.

Reply:
Thanks for explanation!